source/agora/consensus/EnrollmentManager.d

```diff
@@ -924,6 +924,20 @@ public class EnrollmentManager
     {
         this.validator_set.removeAll();
     }
+
+    /***************************************************************************
+
+        Remove the validator from the validator set
+
+        Params:
+            enroll_hash = key for the validator to remove
+
+    ***************************************************************************/
+
+    public void removeValidator (in Hash enroll_hash) @trusted
+    {
+        this.validator_set.remove(enroll_hash);
+    }
```

Comment:
I think we're going with the other approach which is just tagging the disabled validators, and not removing them from the set. This is to support what was implemented in https://github.com/bpfkorea/agora/pull/1421/commits

Reply:
> I think we're going with the other approach which is just tagging the disabled validators, and not removing them from the set. This is to support what was implemented in https://github.com/bpfkorea/agora/pull/1421/commits

Yes, right. The `remove` function will just change the flag of 'active' after the commit being merged.

Reply:
Added `unenrollValidator function` instead of `removeValidator` and new function called `unenroll` into the `ValidatorSet`.